scaleway/resource_lb_frontend.go

```diff
@@ -146,6 +147,11 @@ func resourceScalewayLbFrontend() *schema.Resource {
 											Type: schema.TypeString,
 										},
 									},
+									"http_filter_option": {
```

Comment:
Can we add this to the docs with the example please? This [link](https://developers.scaleway.com/en/products/lb/zoned_api/#http-filter-option-9d5c12) can be useful